bin/install

```diff
@@ -20,6 +22,21 @@ install_helm() {
 
   local download_path="$tmp_download_dir/$(get_filename $version $platform)"
 
+  echo "Checking if download url ${download_url} is valid"
+  local http_status=$(curl -s -o /dev/null -I -w "%{http_code}" "$download_path")
```

Comment:
I think this broke the plugin.

```
$ asdf install helm
Checking if download url https://get.helm.sh/helm-v3.6.3-linux-amd64.tar.gz is valid
Unable to download helm binary
Fast failing
```

Looks like it should be `download_url` instead of `download_path`.

Reply:
Yep, this is breaking our Jenkins builds too